docs/serving/load-balancing/README.md

```diff
@@ -0,0 +1,30 @@
+You can configure load balancing on Knative to place the activator in the request path to act as a load balancer.
+
+This guide explains how you can configure load balancing for your Knative system using the activator.
+
+## Prerequisites
+- Ensure that load balancing using an ingress gateway (for example, Istio or Kourier) is not enabled.
+- Ensure that individual pod addressability is enabled.
+
+## Activator pod selection
+In general, the system will perform best if the number of existing pods is larger than the number of activators, and those numbers divide equally.
+<!--TODO: Add better documentation about what the activator is; explain the components of load balancing; maybe add a diagram-->
+Knative assigns a subset of activators for each revision, depending on the revision size. More revision pods will mean a greater number of activators for that revision. Activators are scaled horizontally, so there may be multiple activators in a deployment.
+
+The activator load balancing algorithm works as follows:
+- If concurrency is unlimited, the request is sent to a random pod.
```

Comment:
This is changed to `best of 2 random choices`.

Reply:
It still doesn't make sense IMHO - a user will want to know what the "2 random choices" refer to. If we mean pods we should just say that, or if we're referring to something else please correct me, but IMHO we should specify where the request is actually sent.